alacritty/src/config/font.rs

```diff
@@ -72,6 +75,22 @@ impl Font {
     }
 }
 
+impl Default for Font {
+    fn default() -> Font {
+        Self {
+            offset: Default::default(),
+            glyph_offset: Default::default(),
+            use_thin_strokes: false,
+            normal: Default::default(),
+            bold: Default::default(),
+            italic: Default::default(),
+            bold_italic: Default::default(),
+            size: Default::default(),
+            builtin_box_drawing: true,
+        }
+    }
+}
```

Comment:
As an alternative, the option could be named `disable_builtin_box_drawing`, then the default of `false` wouldn't require this new impl.

Reply:
I'm generally a in favor of naming config options so the default is `false` rather than `true`, but I don't think this is something even remotely consistently enforced so there's no point being stingy about it when this way around is shorter usually.

**However**, either way the structure of this default impl should be changed. The relavant bit (`builtin_box_drawing`), the only reason why this impl exists at all, is at the far bottom. Making it the last thing you see. Other than that there's `use_thin_strokes: false` which makes it look like someone explicitly deviated from `Default::default()` just to specify this but that's clearly not the case.

The impl should look something like this:

```rust
impl Default for Font {
    fn default() -> Font {
        Self {
            builtin_box_drawing: true,
            use_thin_strokes: Default::default(),
            bold_italic: Default::default(),
            glyph_offset: Default::default(),
            italic: Default::default(),
            offset: Default::default(),
            normal: Default::default(),
            bold: Default::default(),
            size: Default::default(),
        }
    }
}
```

This way it is **obvious** what reason we had to custom-impl `Default`, having all the defaults ordered by length makes it super easy to see at a glance that there's no field accidentally slipping in-between that doesn't use the default value.

Reply:
I changed it to match this exactly but realized the `glyph_offset` line is longer than the `bold_italic` line. Should I flip them?